templates/server/puppetserver/conf.d/puppetserver.conf.erb

```diff
@@ -68,6 +68,11 @@ jruby-puppet: {
     # specified in the Puppet Server HOCON-formatted auth.conf (if false).
     use-legacy-auth-conf: <%= @server_use_legacy_auth_conf %>
 <%- end -%>
+
+<%- if scope.function_versioncmp([@server_puppetserver_version, '2.4']) >= 0 -%>
```

Comment:
the empty line should go below the if and not above, so it's looking nicer in any condition